libfwupdplugin/fu-fmap-firmware.c

```diff
@@ -75,8 +47,9 @@ fu_fmap_firmware_parse(FuFirmware *firmware,
 
 	/* only search for the fmap signature if not fuzzing */
 	if ((flags & FWUPD_INSTALL_FLAG_NO_SEARCH) == 0) {
```

Comment:
This guard still needed? Wasn't it because of CI maybe not having memmem?

Reply:
no, it was the fuzzer would spend hours in stuck in the memmem call trying every possible offset.